### ISSUE TITLE
Download presentation popup window is broken 

### ISSUE THREAD

**Issue by ZhmerenkaSolution** [bigbluebutton/bigbluebutton#19180]
**Describe the bug**
When room administrator enables download of presentation - there is showing up pop-up window message with presentation name and wrong format - .pptx while previously was selected .pdf. And download "linked" text which on pressing redirect in new tab to *domain*/bigbluebutton/undefined which throws:
```
Whitelabel Error Page
This application has no explicit mapping for /error, so you are seeing this as a fallback.

Mon Nov 20 14:30:37 EET 2023
There was an unexpected error (type=Not Found, status=404).
```

**To Reproduce**
1. Persist in room as administrator with uploaded .pptx presentation.
2. Enable download presentation for users (.pdf).
3. Try to download presentation as common user from pop-up window.
4. See error.

**Expected behavior**
Downloading presentation with correct selected extension

**Actual behavior**
Error page, wrong extension in pop-up window.

**Screenshots**
![image](https://github.com/bigbluebutton/bigbluebutton/assets/69646713/ee656cb4-4a24-4cef-8da3-d81c6c9d8edd)
After pressing download:
![image](https://github.com/bigbluebutton/bigbluebutton/assets/69646713/9cf85b45-8140-422d-9760-ffc7b47c046c)

**BBB version:**
BigBlueButton 2.7.2 (463)
Greenlight v3

**Desktop (please complete the following information):**
The issue seems not related to browser

**Smartphone (please complete the following information):**
The issue seems not related to browser

**Additional context**
- The opaque download icon button on presentation works fine.
- Second time enable (disable then again enable) also works fine, but still pop-up window outputs wrong extension.

**Comment by antobinary** [comment]
Thanks for reporting this. It seems the file extension is decisive.

**Comment by antobinary** [comment]
Not able to reproduce on BBB 3.0.0-alpha.4 so not porting the fix ( #19734 ) from BBB 2.7.5 to 3.0.0-alpha.5.
